### PR TITLE
28 create readthe docs

### PR DESCRIPTION
**Pull request by Jiu9Shen**
## Metadata

 - Issue #

- [ ] I feel this PR is ready to be merged.
- [ ] This PR is dependent upon [PR #/ nothing]

Documentation:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

## Testing?

- [ ] I tested on a generic Tripal Site
- [ ] I tested on a KnowPulse Clone
- [ ] This PR includes automated testing

### CODE REVIEW

**Comment by Jiu9Shen** [comment]
Readme.md is not updated yet, I think we can update it after this PR?

**Comment by Jiu9Shen** [comment]
Also, not very confident with writing, so sorry if this ReadtheDocs is bad, but I am happy to make changes

File: docs/install.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+.. code:: bash
+
+  drush pm-enable vcf_filter
+
+
+Now we should able to find this module in Home » Administration » Tripal » Extensions.
+
+.. image:: install.1.menubar.png
+
+We still need to check dependencies and enable the module before make it public.
```
Comment (laceysanderson) [inline docs/install.rst]: At this point the module is ready to use :-) 
```suggestion

```

Comment (laceysanderson) [reply docs/install.rst]: Note: `drush pm-enable vcf_filter` enables the module the same as checking the checkbox. You may want to reword this section to make it flow better?

File: docs/configuration/optional_info.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
+Optinlal information for Adding a file
```
Comment (laceysanderson) [inline docs/configuration/optional_info.rst]:
```suggestion
Optional information for Adding a file
```

File: docs/configuration/required_info.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Required information for Adding a file
+======================================
+
+Only site admins can configure VCF Filter in Home » Administration » Tripal » Extensions » VCF Filter. The following information is required for adding a VCF file:
+  - Aboulute path of the file
```
Comment (laceysanderson) [inline docs/configuration/required_info.rst]:
```suggestion
  - Absolute path of the file
```

File: docs/configuration/test.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
+Test before Publication
+=====================================
+
+An comprehensive test is recommended before make this module public to users.
```
Comment (laceysanderson) [inline docs/configuration/test.rst]:
```suggestion
An comprehensive test of your configuration is recommended before making this module public to users. Some good things to check include:
```

File: docs/install.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+Enable VCF Filter
+-----------------
+After all dependencies are installed and enabled, VCF Filter can be enabled to use in "Home » Administration » Tripal » Modules" of your site.
+
+Also, VCF Filter can be enabled by drush command:
+
+.. code:: bash
+
+    drush pm-enable vcf_filter
```
Comment (laceysanderson) [inline docs/install.rst]:
```suggestion
    drush pm-enable vcf_filter

This command will enable the module after which we should able to find it in Home » Administration » Tripal » Extensions.

.. image:: install.1.menubar.png
```

File: docs/install.rst [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+The module is availabe as one repository for `Pulse Bioinformatics, University of Saskatchewan <https://github.com/UofS-Pulse-Binfo>`_ on GitHub. Recommended method of downloading and installation is using git:
+
+.. code:: bash
+
+  cd [your drupal root]/sites/all/modules
+
+  git clone https://github.com/UofS-Pulse-Binfo/vcf_filter.git
+
+The commands from above will download the module into the specified directory. And we should able to find this module in Home » Administration » Tripal » Extensions.
```
Comment (laceysanderson) [inline docs/install.rst]:
```suggestion
```

File: docs/install.rst [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+.. image:: install.1.menubar.png
```
Comment (laceysanderson) [inline docs/install.rst]:
```suggestion
```

**Pull request merged**